src/redux/reducers/bookmarksReducer.js

```diff
@@ -18,6 +18,9 @@ function removeBookmark (state, routeId) {
 }
 
 export default function reducer (state = initialState, action) {
+    if (!Map.isMap(state)) {
```

Comment:
Nop, de to linjene under hadde feil indent, så jeg var ikke sikker på om du mente at if settningen også hadde feil ;)